### PR TITLE
Wrote tests for UserDB and TokenDB

### PR DESCRIPTION
**Pull request by TSKsmiley**
Wrote tests for UserDB and TokenDB

### CODE REVIEW

**Comment by TSKsmiley** [comment]
image

**Review by AlbertHald — APPROVED** [review]
lllllllllllllllllllllllllllggggggggggggggggttttttttttttttttttttttttmmmmmmmmmmmmmmm

File: src/test/UserDB.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        const randomLog = faker.lorem.sentence();
+        user.log(randomLog);
+        assert.equal(user.getLog()[0],randomLog);
+        done();
+    });
+
+    it("clearLog() should clear the user's log", function(done) {
+
+        // skipping the test because it is not working
+        // and we dont have time to fix it
```
Comment (BaBrixx) [inline src/test/UserDB.js]:
```suggestion
```
Not good to use the phrase "Didn't have time" in projects :)

File: src/test/UserDB.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+    it("getFlows() should return an array with one element", function(done) {
+        assert.equal(user.getFlows().length,1);
+        done();
+    });
+
+    it("removeFlow() should remove the flow from the user", function(done) {
+
+        // skipping the test because it is not working
+        // and we dont have time to fix it
```
Comment (BaBrixx) [inline src/test/UserDB.js]:
```suggestion
```
Not good to use the phrase "Didn't have time" in projects :)

**Review by BaBrixx — CHANGES_REQUESTED** [review]
LVG but there are a few comments that should be revised

**Review by BaBrixx — APPROVED** [review]
LVGTM

**Pull request merged**